src/config.rs

```diff
@@ -340,6 +345,19 @@ pub enum Config {
     VerifiedOneOnOneChats,
 }
 
+impl Config {
+    /// Whether the config option is synced across devices.
+    ///
+    /// This must be checked on both sides so that if there are different client versions, the
+    /// synchronisation of a particular option is done or not done both ways.
```

Comment:
```suggestion
    /// Note that receivers of a config value also need to check if a key can be synced. 
    /// Some settings (e.g. Avatar path) could otherwise
    /// lead to exfiltration of files from a receiver's device
    /// if we assume an attacker to have control of a device in a
    /// multi-device setting.
```

Reply:
A good note, but security considerations are not the only reason, so i left the existing part of the comment also. The idea is that the synchronisation should be bidirectional even for different versions of clients, if some config option is synced from device A to B, it also should be synced from B to A.